### ISSUE TITLE
key-naming-case only checks the first key in a call; later keys are silently ignored

### ISSUE THREAD

**Issue by AustinAbro321** [go-simpler/sloglint#132]
With `key-naming-case` set (e.g. `camel`), sloglint only validates the first key-value pair in a log call. If the first key is valid, subsequent invalid keys in the same call are silently skipped.

### Reproduction
1.  Create the following go file
```go
package main

import "log/slog"

func main() {
	l := slog.Default()

	// BUG: only the first key is checked. "bad-key" is NOT flagged because
	// the preceding key "good" is valid camelCase.
	l.Info("not flagged",
		"good", "a",
		"bad-key", "b")

	// Control: when the invalid key comes first, sloglint catches it.
	l.Info("flagged",
		"bad-key", "b",
		"good", "a")
}
```
2. Run: `sloglint -key-naming-case=camel .`
```bash
/tmp/sloglint-bug-repro/main.go:16:3: keys should be written in camelCase
````
It only gives one error message, for the second slog statement.

**Comment by tmzane** [comment]
Already fixed in `main`, closing. Thanks!

**Comment by AustinAbro321** [comment]
That's great! A release will be much appreciated when you get the chance
